client/src/featureform/register.py

```diff
@@ -2572,6 +2572,43 @@ def apply(self):
         else:
             state().create_all(self._stub)
 
+    def continue_waiting(status, timeout, time_waited):
```

Comment:
This sounds like an action instead of a check. Should be name something more like `should_wait`